samples/peptide.py

```diff
@@ -39,7 +34,7 @@
 
 # Peptide parameters
 
-sequence = 'nEEEEEc'
+sequence = 'EEEEDDDD'
```

Comment:
@davidbbeyer the reason is that the termini were actually not parametrized in `parameters/peptides/Lunkad2021.json`. In earlier versions of this script, this was bypassed by adding a generic parametrization for the non-parametrized aminoacid particles. This was a convenient solution to allow to run any peptide sequence, but not really a good practice (since we were basically teaching people to mix parameters instead of thinking about what should be a good parametrization). With @mariusaarsten we have done a more complete parametrization of the amino acids which he will add in his master thesis and we will also provide in pyMBE so we can have a first complete set of parameters for all aminoacids.